bliss/transforms.py

```diff
@@ -21,3 +67,8 @@ def pixelwise_norm_source(img):
 def log_transform(img):
     """Perform pixel-wise log transformation of full image."""
     return torch.log(img)
+
+
+def tanh_mod(img, scale, shift):
```

Comment:
Did this help? If not, probably better to remove it. Better not to keep it around "just in case" because it's extra code to maintain.

Reply:
Nope, we should remove that.

Reply:
By 'this', what functions are you referring to exactly? Just the hyperbolic tangent?

Reply:
Yep